### DISCUSSION TITLE
Scheduler instance to only manage cluster

### DISCUSSION THREAD

**Discussion by recepaslan** [quartznet/quartznet#1057] [Q&A] [answered]
Hi, 

Consider the following infrastructure setup. 1 one application and 2 background services. While background services execute jobs, web application only manages jobs and schedules. 

[Documentation ](https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/advanced-enterprise-features.html#clustering) indicates that all instance to have the same setup but when a web application initiates an instance it starts to take a load of background services.

How can we configure quartz instance in the web application that it will not take any load but can schedule, reschedule, etc jobs?

Thank you.

**Comment by lahma** [comment, marked as answer]
You need to configure the scheduler with ZeroSizeThreadPool and skip starting it. It won't process any work then.

**Reply by recepaslan** [reply]
In my case, I have used `quartz.config` with configuration `quartz.threadPool.type = Quartz.Simpl.ZeroSizeThreadPool`, which worked very well.

Appreciated the help.
